gulpfile.babel.js

```diff
@@ -551,27 +550,6 @@ gulp.task( 'languages:extract', function( done ) {
 /*
  * Gutenpack!
  */
-gulp.task( 'gutenpack', function() {
-	return gulp.src( [ '**/*/*block.jsx', ...alwaysIgnoredPaths ] )
-		.pipe( babel( {
-			plugins: [
-				[
-					'transform-react-jsx', {
-						pragma: 'wp.element.createElement'
-					}
-				]
-			],
-		} ) )
-		.on( 'error', function( err ) {
-			log( colors.red( err ) );
-		} )
-		.pipe( gulp.dest( './' ) );
-} );
-
-gulp.task( 'gutenpack:watch', function() {
-	return gulp.watch( [ '**/*/*block.jsx', ...alwaysIgnoredPaths ], [ 'gutenpack' ] );
-} );
-
 gulp.task( 'gutenpack:jetpack-blocks', function() {
```

Comment:
Feels like this one could be named just `gutenpack` now, right?

Reply:
Or something about blocks or Gutenberg assets.

Reply:
Y'all good with `gutenpack:jetpack-blocks`?

Reply:
(`jetpack` is kinda implicit anyway, since this is the Jetpack codebase :smile: )